packages/llvm/llvm.3.9/files/install.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+
+make=$1
+libdir=$2
+
+# Create build dir
+mkdir build
+cd build
+
+# Configure
+cmake ..
+#    -DLLVM_BUILD_LLVM_DYLIB:BOOL=ON \
+#    -DLLVM_LINK_LLVM_DYLIB:BOOL=ON \
+#    ..
+
+# Build
+grep "^ocaml_llvm.*/fast:$" Makefile | tr ":\n" " " | xargs "$make"
```

Comment:
Actually, you could replace your horrible `make` hack with this:
```
--- a/cmake/modules/AddOCaml.cmake
+++ b/cmake/modules/AddOCaml.cmake
@@ -174,6 +174,8 @@ function(add_ocaml_library name)
     add_dependencies("ocaml_${name}" "${llvm_lib}")
   endforeach()
 
+  add_dependencies("ocaml_all" "ocaml_${name}")
+
   set(install_files)
   set(install_shlibs)
   foreach( ocaml_output ${ocaml_outputs} )
@@ -205,3 +207,5 @@ function(add_ocaml_library name)
       VERBATIM)
   endforeach()
 endfunction()
+
+add_custom_target("ocaml_all")
```
and then just `make ocaml_all`. I will push this into upstream shortly.